lib/view_component/use_helpers.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module ViewComponent::UseHelpers
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def use_helpers(*args)
+      args.each do |helper_mtd|
+        class_eval(<<-RUBY, __FILE__, __LINE__ + 1)
+          def #{helper_mtd}(*args, &block)
```

Comment:
```suggestion
          def #{helper_method}(*args, &block)
```

Similar to the above, can we avoid shorthand?